metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -123,8 +123,14 @@ class CliReport(Report):
     py_version: str = sys.version
     py_exec_path: str = sys.executable
     os_details: str = platform.platform()
+    _peek_memory_usage: int = 0
 
     def compute_stats(self) -> None:
+        mem_usage = psutil.Process(os.getpid()).memory_info().rss
```

Comment:
Minor thing -> We call psutil.Process(os.getpid()).memory_info().rss 2 times in this method to fetch memory info. Why not fetch it once?